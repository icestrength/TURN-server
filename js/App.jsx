// @flow

import React from "react";
import { BrowserRouter, Route, Switch } from "react-router-dom";

const FourOhFour = () => <h1>404402</h1>;

const App = () => (
  <BrowserRouter>
    <div className="app">
      <Switch>
        <Route component={FourOhFour} />
      </Switch>
    </div>
  </BrowserRouter>
);

export default App;
