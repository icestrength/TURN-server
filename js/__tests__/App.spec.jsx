import React from "react";
import { shallow } from "enzyme";
import App from "../App";

test("App renders correctly", () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
