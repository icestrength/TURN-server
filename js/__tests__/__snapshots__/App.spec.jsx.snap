// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<BrowserRouter>
  <div
    className="app"
  >
    <Switch>
      <Route
        component={[Function]}
      />
    </Switch>
  </div>
</BrowserRouter>
`;
